packages/site/src/components/PagerView.tsx

```diff
@@ -44,21 +53,29 @@ export function View({ color }: { color: string }) {
   };
 
   return (
+    // @ts-ignore
```

Comment:
There was a weird circular reference error being thrown here that I couldn't figure out the cause. Possibly the `css` prop type conflicting with react-spring.